be/src/main/java/com/issuetracker/mapper/JacksonLocalDateTimeConfig.java

```diff
@@ -0,0 +1,42 @@
+package com.issuetracker.mapper;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+@Configuration
+public class JacksonLocalDateTimeConfig {
+    @Bean
+    public SimpleModule jsonMapperJava8DateTimeModule() {
+        SimpleModule module = new SimpleModule();
+
+        module.addSerializer(LocalDateTime.class, new JsonSerializer<LocalDateTime>() {
+            @Override
+            public void serialize(
+                    LocalDateTime localDateTime, JsonGenerator jsonGenerator, SerializerProvider serializerProvider)
+                    throws IOException {
+                jsonGenerator.writeString(DateTimeFormatter.ofPattern("yyyy-MM-dd kk:mm:ss").format(localDateTime));
+            }
+        });
```

Comment:
여기부분은 람다식으로 좀 더 깔끔하게 작성하는건 어떻게 생각하세요?

Reply:
JsonSerializer가 함수형인터페이스가 아닌 추상클래스여서 람다식으로 작성되지 않는것같은데요..ㅠㅠ 혹시 다른 방법이 있을까요?

Reply:
람다식이 아니고 ObjectMapper를 따로 구현해서 체이닝이 가능하네요.

```suggestion
  @Bean
  public ObjectMapper objectMapper() {
    return new ObjectMapper()
        .registerModule(new JavaTimeModule()
            .addSerializer(LocalDate.class, new LocalDateSerializer(DateTimeFormatter.ofPattern("데이트타입")))
            .addSerializer(LocalDateTime.class, new LocalDateTimeSerializer(DateTimeFormatter.ofPattern(
                "데이트시간타입")));
    }
```